.golangci.yml

```diff
@@ -33,6 +33,9 @@ linters-settings:
       - whyNoLint
   gocyclo:
     min-complexity: 15
+  godox:
+    keywords:
+      - FIXME
```

Comment:
Unrelated to this PR?

Reply:
It's related :smile: because during the dev I put a lot of FIXME and I forgot to remove them, so I enabled godox.
So it's unrelated but related.